src/client/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -14,6 +12,9 @@ internal class AadAuthority : Authority
         public const string DefaultTrustedHost = "login.microsoftonline.com";
         public const string AADCanonicalAuthorityTemplate = "https://{0}/{1}/";
 
+        private const string TokenEndpointTemplate = "{0}oauth2/v2.0/token";
+        private const string AuthorizationEndpointTemplate = "{0}oauth2/v2.0/authorize";
```

Comment:
since these are the same for both AAD and ADFS, can they be moved to a constants class?

Reply:
These are slightly different between AAD and ADFS, notice the "v2.0" part.